js/front/dataset/index.js

```diff
@@ -64,13 +72,16 @@ new Vue({
         /**
          * Display a resource or a community ressource in a modal
          */
-        showResource(id, e, isCommunity) {
-            // Ensure edit button work
-            if ([e.target, e.target.parentNode].some(el => el.classList.contains('btn-edit'))) return;
-            e.preventDefault();
+        showResource(id, isCommunity) {
             const attr = isCommunity ? 'communityResources' : 'resources';
             const resource = this.dataset[attr].find(resource => resource['@id'] === id);
-            this.$modal(ResourceModal, {resource});
+            const communityPrefix = isCommunity ? '-community' : '';
+            location.hash = `resource${communityPrefix}-${id}`;
+            const modal = this.$modal(ResourceModal, {resource});
+            modal.$on('modal:closed', () => {
+                // prevent scrolling to top
```

Comment:
Is this behavior common to all modals on close ?
Then maybe be we should issue a fix in a separate PR for this ?

Reply:
Nope, it's for this one only: since I set the `location.hash` with the current resource hash, I need to unset it when closing the modal. The `#_` is just a trick to avoid the top scroll that would be triggered with `#`.